dali/test/python/operator_2/test_to_decibels.py

```diff
@@ -211,4 +211,4 @@ def test_operator_natural_logarithm():
 
 @raises(RuntimeError, glob="`reference` argument can't be zero")
 def test_invalid_reference():
-    NLDaliPipeline("cpu", None, 1, reference=0.0).build()
+    NLDaliPipeline("cpu", None, 1, reference=0.0)
```

Comment:
The pipeline is not used now; please restore .build().
```suggestion
    NLDaliPipeline("cpu", None, 1, reference=0.0).build()
```

Reply:
done